src/helpers/makeEdgeSizes.js

```diff
@@ -0,0 +1,10 @@
+// return the Grommet edgeSize object(in themes/global)
+// using the specified base unit(base) and range values(min, max)
+
+export const makeEdgeSizes = (base, min, max) =>
```

Comment:
```suggestion
export const getEdgeSizes = (base, min, max) =>
```

Reply:
It technically sets (generates new values) for Grommet `edgeSize` in themes/global, so renamed as `setEdgeSizes` instead, thank you!